scripts/oneoff/insert-g8-framework-results.py

```diff
@@ -0,0 +1,33 @@
+"""
+For each G-Cloud 8 application this script:
+ * gets the supplier declaration
+ * checks whether it is a PASS, FAIL or DISCRETIONARY
+ * updates the `supplier_frameworks` entry for the application accordingly, using the API:
+   - on_framework=true if declaration OK and at least one completed service
+   - on_framework=false if declaration FAILed or there are no completed services
+   - on_framework=NULL (no call to API) if declaration DISCRETIONARY and at least one completed service
+
+Usage:
+    scripts/insert-g8-framework-results.py <stage> --api-token=<data_api_token>
+
+Example:
+    python scripts/insert-dos-framework-results.py dev --api-token=myToken
```

Comment:
I think this is wrong.  Some people might get confused and spend 20 minutes on the wrong thing :/